core-web/apps/dotcms-ui/src/app/portlets/dot-categories/dot-categories-create-edit/dot-categories-create-edit.component.scss

```diff
@@ -0,0 +1,23 @@
+@use "variables" as *;
+
+:host ::ng-deep {
+    .caategories-create__tab-container {
+        display: flex;
+        flex: 1;
+        flex-direction: column;
+    }
+
+    .p-tabview-nav {
+        padding: 0 $spacing-4;
+        background: $white;
+    }
+
+    .p-tabview-panel {
+        padding: 0 $spacing-4;
+        padding-top: $spacing-4;
+    }
+}
+
+dot-portlet-base {
+    overflow: scroll;
+}
```

Comment:
`overflow: scroll;` always display scrollbars (vertical and horizontal) whether or not any content is clipped. Is that what we need?

Reply:
no, we needed a horizontal scrollbar when content is clipped and exceed the view. I change it to this `overflow-x: auto;` is it ok?